.gitignore

```diff
@@ -1,3 +1,5 @@
+.envrc
+.venv
```

Comment:
Also curious: are you using poetry with `virtualenvs.in-project` set to True, and if so, why?

Reply:
See #71

Reply:
Not really. I actually use:
```
$ cat ~/.config/pypoetry/config.toml 
[virtualenvs]
create = false
```

Which means that poetry does not manage my virtualenvs at all. I started doing this because I wanted to have more control over which python I will be using on each project, while poetry only uses the currently active python (at least that's what it used to do in the past). In the poetry bug tracker there is a rather long thread that discusses the issue: https://github.com/python-poetry/poetry/issues/522

In a nutshell, what I want from poetry is a reliable dependency resolver. And it does a great job at that.
In the end I found it easier/simpler to manage the environment creation/activation/setting up using other tools (direnv included).

WRT to having the virtualenv inside the project, I do it because I find it convenient to have the source code of the libraries I use easily reachable/searchable. "grep"-ing through the source code is still possible by using a .gitignore-aware tool like [ag](https://github.com/ggreer/the_silver_searcher)

Reply:
Interesting